src/BreakfastSerializer/ConfigurableProperty.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace BDBStudios\BreakfastSerializer;
+
+use Symfony\Component\Yaml\Exception\ParseException;
+use Symfony\Component\Yaml\Yaml;
+
+/**
+ * Class ConfigurableProperty
+ * @package BDBStudios\BreakfastSerializer
+ */
+trait ConfigurableProperty
+{
+
+    /**
+     * @var string
+     */
+    protected $configurationPath;
+
+    /**
+     * @var array
+     */
+    protected static $configurationData;
+
+    /**
+     * @param string $pathName
+     * @return IsConfigurable
+     */
+    public function setConfigurationPath($pathName = './config')
```

Comment:
Possibly, though I wanted it to run without dicking about with a bootstrap, should probably include the default directory as well...